api/external/applications/applications.proto

```diff
@@ -211,58 +356,92 @@ message GetDeleteDisconnectedServicesConfigReq{}
 message UpdateDeleteDisconnectedServicesConfigRes {}
 
 message Service {
+  // The habitat supervisor ID
```

Comment:
Including a trailing period or not on these comments changes whether these will become the `title` or `description` (if I remember right) when converting the protos to swagger. We've settled on a convention of including the trailing period./